pyparsing/core.py

```diff
@@ -2299,7 +2301,7 @@ def must_skip(t):
             def show_skip(t):
                 if t._skipped.as_list()[-1:] == [""]:
                     t.pop("_skipped")
-                    t["_skipped"] = "missing <" + repr(self.anchor) + ">"
+                    t["_skipped"] = f"missing <{self.anchor}>"
```

Comment:
Used to show `repr(self.anchor)`  - should change to `{self.anchor!r}`"?

Reply:
This change was so that an [existing unit test](https://github.com/pyparsing/pyparsing/blob/81605f7ddc3a6217fffc57a71a93c72cc83e2d41/tests/test_unit.py#L1620) would pass. I'm still hazy on the what/why of `SkipTo` in this context - should the test be changed instead?